InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -301,117 +398,11 @@ public void teleopInit() {
-                } else if (climbSubsystem.isRewindEnabled()) {
-                    climbSubsystem.pitRewind(oi.pitRewindLeft(), oi.pitRewindRight());
-                }
-            }
-
-        }
         //////////////////////////////////////////////////////////////////////////////
         // Shooter Subsystem
-
         if (config.enableShooterSubsystem) {
-            SmartDashboard.putNumber("BallManagementSubsystem/Output Percent", 50);
-
-            if (oi.feeder()){
-                shooterSubsystem.spinFeeder();
-            } else {
-                shooterSubsystem.stopSpinningFeeder();
-            }
-
-            // Spin up on pressing [spinUp] and auto aim on pressing [aimBot]
-            if (oi.spinUp()) {
-                shooterSubsystem.startSpinningUp();
-            } else if (oi.operatorAimBot()) {
-                shooterSubsystem.autoAim();
-            } else {
-                shooterSubsystem.stopSpinningUp();
-                shooterSubsystem.stopAutoAim();
-            }
-
-            // Fire on pressing [fire]
-            if (oi.fire()) {
-                shooterSubsystem.spinBMS();
-            } else {
-                shooterSubsystem.holdFire();
-            }
-
-            // Rotate the turret with [manualAzimuthAxis]
-            if (Math.abs(oi.manualAzimuthAxis()) >= config.shooter.manualAzimuthDeadband
-                    || Math.abs(oi.manualElevationAxis()) >= config.shooter.manualElevationDeadband) {
-                shooterSubsystem.rotate(oi.manualAzimuthAxis(), oi.manualElevationAxis());
-            } else {
-                shooterSubsystem.rotate(0, 0);
-            }
-
-            if (oi.zero()) {
-                shooterSubsystem.rotateToDeg(0, 0);
-            }
-
-            if (oi.nextPositionElevation()) {
-                shooterSubsystem.nextPositionElevation();
-            } else if (oi.lastPositionElevation()) {
-                shooterSubsystem.lastPositionElevation();
-            } else {
-                shooterSubsystem.resetPositionElevationSwitcher();
-            }
-
-            if (oi.setElevationToDashboardNumber()) {
-                shooterSubsystem.rotateToDeg(shooterSubsystem.getTargetAzimuthDeg(),
-                        SmartDashboard.getNumber(shooterSubsystem.getName() + "/Dashboard Elevation Target", config.shooter.DEFAULT_ELEVATION_TARGET_DEG));
-            }
-        }
-
-        // //////////////////////////////////////////////////////////////////////////////
-        // // SpinnyBoi Subsystem
-        if (config.enableSpinnyboiSubsystem) {
-            if (oi.spinnyBoiForward()) {
-                spinnyBoiSubsystem.forward();
-            } else if (oi.spinnyBoiBackward()) {
-                spinnyBoiSubsystem.backward();
-            } else {
-                spinnyBoiSubsystem.off();
-            }
+            SmartDashboard.putNumber("BallManagementSubsystem/Output Percent", 50);            

```

Comment:
Heh, LAST STATEMENT STANDING.

Reply:
Didn't know what to do with it and we haven't moved to oblog yet so just let it be lol